### PR TITLE
C105b

### PR DESCRIPTION
**Pull request by raj8789**
I Have generated code coverage report in Index.html file

### CODE REVIEW

**Review by PoojaMohite-Khoros — CHANGES_REQUESTED** [review]
REmove unnecessary files
Also generate report and post in channel

File: pom.xml [hunk 1]
```diff
@@ -41,9 +41,30 @@
             <version>4.12</version>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <version>0.8.7</version>
+        </dependency>
```
Comment (anasuya1117) [inline pom.xml:48]: Why is this needed

File: target/generated-test-sources/index.html [hunk 2]
```diff
@@ -0,0 +1,197 @@
+
```
Comment (anasuya1117) [inline target/generated-test-sources/index.html]: no need to include this in git, please remove it

**Review by anasuya1117 — APPROVED** [review]
Please address the comments and merge
